src/wp-admin/css/revisions.css

```
@@ -25,8 +25,8 @@
 	position: fixed;
 	top: 0;
 	height: 82px;
-	background: #fff;
-	box-shadow: 0 1px 3px rgba(0, 0, 0, 0.1);
+	background-color: var(--wp-admin--revisions--controls--background--pinned);
+	box-shadow: 0 1px 3px var(--wp-admin--revisions--controls--box-shadow--pinned);
```

Let's stick with the `--wp-admin--shadow` property for transparent black values, it's used with [functional rgb notation](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb()) like this

```suggestion
	box-shadow: 0 1px 3px rgb(var(--wp-admin--shadow) / 0.1);
```